debezium-core/src/main/java/io/debezium/connector/SnapshotRecord.java

```diff
@@ -15,8 +15,8 @@
  */
 public enum SnapshotRecord {
     TRUE, // record is from snapshot is not the last one
-    FALSE, // record is from snapshot is the last record generated in snapshot phase
```

Comment:
Ha, good catch. While you're at it, could you convert those comments into proper JavaDoc:

    /**
     * record is from snapshot but not the last one
     */
    TRUE

Reply:
OK, I have done it and rebased.